tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import sys
+import os
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
```

Comment:
What's the purpose of this?

Reply:
This file stores test fixtures that can be automatically imported and used in multiple test modules.

https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions

Reply:
They sys.path.insert is so I can add the directory above the one I'm testing to the environment path, so I can import inquestlabs. Without these lines...

```python
import sys
import os
sys.path.insert(1, os.path.join(sys.path[0], '..'))
```
Allows the line below it to be a valid import:

`from inquestlabs import inquestlabs_api`